compiler/src/iree/compiler/Codegen/LLVMCPU/test/tile-root-fuse-consumer-producer.mlir

```diff
@@ -117,3 +117,40 @@ func.func @dequant_avgpool(%arg0: tensor<1x320x65x65xi8>) -> tensor<1x320x1x1xf3
 // CHECK-REDUCTION:               }
 // CHECK-REDUCTION:             }
 // CHECK-REDUCTION:           }
+
+// -----
+
+#pipeline_layout = #hal.pipeline.layout<constants = 2, bindings = [#hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, Indirect>], flags = Indirect>
+#translation = #iree_codegen.translation_info<pipeline = Mmt4dTilingExpert>
```

Comment:
nit: we do not need the translation_info in the test. (I did not notice it in the previous review, my bad.)